### ISSUE TITLE
Export Stellar private key w/o desktop or mobile apps

### ISSUE THREAD

**Issue by sdpngd** [keybase/keybase-issues#3634]
Hello, I have a keybase account that was awarded with a few Stellar - I got this notification email. The issue is that I have wiped my device where I had installed the desktop app; so now, I can only log into the webui to manage my keybase account. I understand that in order to provision a new device, I either need my already provisioned device (not possible as it is already wiped) or I need to reset my account and start over. Now, when I tried to go for the reset using the "forgot passphrase" option in the webui, I was alerted to the fact that doing so would make me lose the Stellar that have been awarded to my account. And upon further reading, I understand that the only way to export the private key for the Stellar wallet is via the desktop app.

So to sum up, is there any way I can export my keys without a desktop app and then proceed to reset my account to provision a new device? Or the only option is to forfeit the awarded Stellar and go for a reset? Appreciate your guidance.

**Comment by Brianetta** [comment]
The active devices on your sigchain are the only tools that can decrypt your Stellar secret key, in the same way that they are the only tools that can provision a new device. They were lost when you wiped your device, which is, I'm sorry to say, the moment that you permanently lost your wallet.

**Comment by hex17or** [comment]
Was afraid of this. Thanks for your reply nevertheless.
